src/app/frontend/replicasetdetail/replicasetdetail.scss

```diff
@@ -26,7 +27,8 @@ $replicasetdetails-table-cell: #777;
 
 .kd-replicasetdetail-sidebar {
   background-color: $replicasetdetails-sidebar-bg;
-  min-width: 315px;
+  max-width: $replicasetdetails-sidebar-width;
```

Comment:
I believe this caused a bug:
![selection_008](https://cloud.githubusercontent.com/assets/1159999/12195303/750d3e56-b5f6-11e5-8e1c-829d7b71dc58.png)